bot/exts/moderation/modlog.py

```diff
@@ -522,14 +521,32 @@ async def log_cached_deleted_message(self, message: discord.Message) -> None:
                 f"**Message ID:** `{message.id}`\n"
                 f"**Sent at:** {format_dt(message.created_at)}\n"
                 f"[Jump to message]({message.jump_url})\n"
-                "\n"
             )
 
+        # If the message is a reply, add the reference to the response
+        if message.reference is not None and message.reference.resolved is not None:
+            resolved_message = message.reference.resolved
+
+            if isinstance(resolved_message, discord.DeletedReferencedMessage):
+                # Reference is a deleted message
+                reference_line = "**In reply to:** Deleted Message"
```

Comment:
Don't you need to add a newline here?
```suggestion
                reference_line = "**In reply to:** Deleted Message\n"
```